documentation/guides/record-tests/README.md

```diff
@@ -35,13 +35,13 @@ You can record a test in any local browser. TestCafe detects browsers on the loc
 
 ![Record Congiruation Menu](../../../images/guides/record-configuration-list.png)
 
-You can modify record configuration options by clicking the ![Settings button](../../../images/guides/settings-icon.png) button and using the [Record Configuration](../../user-interface/record-configuration-dialog.md) dialog.
+Click the ![Settings button](../../../images/guides/settings-icon.png) button and use the [Record Configuration](../../user-interface/record-configuration-dialog.md) dialog to modify record configuration options.
 
 ![Record Configuration Dialog](../../../images/user-interface/dialogs/record-configuration-dialog.png)
 
 ## Create Fixtures
 
-Recorded tests must be organized into categories called **fixtures**. A fixture contains tests for a single target URL, and you usually have one fixture per website page.
+Recorded tests are organized into categories called **fixtures**. A fixture contains tests for a single target URL, and you usually have one fixture per website page.
```

Comment:
Tests aren't organized. The user must organize them into a fixture, namely he/she should first create a fixture, then start record a test.

Reply:
Recorded tests **should** be organized into categories called fixtures. A fixture contains tests for a single target URL, and you usually have one fixture per website page.